ui/shared/components/table/RnaSeqJunctionOutliersTable.jsx

```diff
@@ -72,7 +72,26 @@ const OTHER_SPLICE_COLUMNS = [
     }
   )),
   {
-    name: 'rareDiseaseSamplesWithJunction',
+    name: 'counts',
```

Comment:
not a blocker for this PR, but would you be open to including screenshots/recordings of screen interactions in the PR description when there are UI changes? I'm curious to see what the changes look like but I'll likely only interact with these changes in seqr if I remember to after this gets merged

Reply:
yeah that seems very reasonable!